deployment/chainloop/values.yaml

```diff
@@ -150,6 +150,15 @@ controlplane:
     grpc: 9000
     metrics: 5000
 
+  ##
+  ## TLS configuration
+  ##
+  tls:
+    ## @param controlplane.tls.existingSecret Existing secret with TLS certificates. The secret must contains 2 keys: tls.crt and tls.key respectively containing the certificate and private key.
+    ## NOTE: When it's set it will disable secret creation.
+    ##
+    existingSecret: ""
+
   ## @param controlplane.tlsConfig.secret.name name of a secret containing TLS certificate to be used by the controlplane grpc server.
   tlsConfig:
```

Comment:
I remove this from the values.yaml file, it's a deprecated option, we do not need to document it. Note that once you remove it you'll need to update your helper to check for the nil struct.